.gitignore

```diff
@@ -0,0 +1,16 @@
+# Xcode
+# Picked from https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore
+
+xcuserdata/
+build/
+DerivedData/
+*.xcworkspace
+
+# Swift
+.build/
+#Pods/
+#Carthage/Build
+#Dependencies/
+fastlane/report.xml
```

Comment:
Why did you add references to fastlane ? As i known, we don't use fastlane at the moment

Reply:
Copy paste of .gitignore template x)

Reply:
Update .gitignore ;)